docs/sagas.md

```diff
@@ -0,0 +1,239 @@
+---
+id: sagas
+title: Sagas
+---
+
+A saga describes a long running process as a sequence of events.
+
+## Sagas Overview
+
+You can define a saga as a set of event handler functions. Each such function runs in response to a specific event and can do the following:
```

Comment:
Change "Each such function" to "Each function".

Reply:
done